frontend/src/scenes/trends/PersonsModal.tsx

```diff
@@ -200,28 +199,37 @@ export function PersonsModal({
-                                                        breakdownValue={
-                                                            dataPoint.breakdown_value === ''
-                                                                ? 'None'
-                                                                : dataPoint.breakdown_value?.toString()
-                                                        }
-                                                        showCountedByTag={showCountedByTag}
-                                                        hasMultipleSeries={hasMultipleSeries}
-                                                    />
-                                                </Select.Option>
-                                            ))}
-                                        </Select>
-                                    </div>
+                                    <LemonSelect
+                                        fullWidth
+                                        className="mb-2"
+                                        value={people.seriesId.toString()}
+                                        onChange={(_id) =>
+                                            typeof _id === 'string' ? switchToDataPoint(parseInt(_id, 10)) : null
+                                        }
+                                        options={
+                                            people.crossDataset.reduce(
+                                                (acc, dataPoint) => ({
+                                                    ...acc,
+                                                    [`${dataPoint.id}`]: {
+                                                        label: (
+                                                            <InsightLabel
+                                                                seriesColor={getSeriesColor(dataPoint.id)}
+                                                                action={dataPoint.action}
+                                                                breakdownValue={
+                                                                    dataPoint.breakdown_value === ''
+                                                                        ? 'None'
+                                                                        : dataPoint.breakdown_value?.toString()
+                                                                }
+                                                                showCountedByTag={showCountedByTag}
+                                                                hasMultipleSeries={hasMultipleSeries}
+                                                            />
+                                                        ),
+                                                    },
+                                                }),
+                                                {}
+                                            ) as LemonSelectOptions
```

Comment:
@pauldambra relevant to LemonSelect API - the typing is really complex it seems and doesn't always pick up the correct type...